src/Sulu/Bundle/MediaBundle/Behat/MediaContext.php

```diff
@@ -153,7 +153,7 @@ public function iShouldSeeItems($count)
      * Return the last media collection that was created
      * in this context
      *
-     * @return Sulu\Bundle\MediaBundle\Entity\Collection
+     * @return \Sulu\Bundle\MediaBundle\Entity\Collection
```

Comment:
well, wihout the slash the namespace would be resolved as `/Sulu/Bundle/MediaBundle/Behat/Sulu/Bundle/MediaBundle/Entity/Collection`.

As discussed somewhere else we should just `use` the entity, e.g. `use Sulu\Bundle\MediaBundle\Entity\Collection`